### DISCUSSION TITLE
Trying to use the 'Custom object detection model example' to vectorize the trajectory of detected objects in 2D.

### DISCUSSION THREAD

**Discussion by runhuayao** [mikel-brostrom/yolo_tracking#935] [Ideas]
I want to use “Custom object detection model example”，use (x1, y1, x2, y2) coordinates of detected objects and combine them with object tracking to create a 2D trajectory coordinate map of detected objects. My idea is to create a 'detect_objects' function in the first '...' that takes my trained model, such as yolov8s.pt, as input to traverse the image and return a numpy array so that it can be directly connected to the author's sample code. The second '...' is where I plan to write the code to generate the 2D trajectory coordinate map of detected objects once the above code is confirmed to be working.
But I am not sure is it right？

**Comment by mikel-brostrom** [comment]
Sounds good to me 😄. Make sure that the input to the tracker is exacly what is specified under the README

**Reply by runhuayao** [reply]
Wow, thank you for the encouragement. I will try my best and take note of the input you mentioned.
